cluster/gce/gci/configure-helper.sh

```diff
@@ -484,7 +484,7 @@ function start-kubelet {
     flags+=" --enable-debugging-handlers=false"
     flags+=" --hairpin-mode=none"
     if [[ ! -z "${KUBELET_APISERVER:-}" && ! -z "${KUBELET_CERT:-}" && ! -z "${KUBELET_KEY:-}" ]]; then
-      flags+=" --api-servers=https://${KUBELET_APISERVER}"
+      flags+=" --api-servers=https://${MASTER_RESERVED_IP}"
```

Comment:
The previous check makes sure that `KUBELET_APISERVER` is not empty, but now you are using a different variable. You should update the check.